server/src/Game/Game.js

```diff
@@ -86,7 +86,7 @@ export default class Game {
         const dt = this._tick() / 1000
         Object.values(this.players).forEach(player => {
             player.socket.emit(EVENTS.SERVER_TICK, dt)
-            const pchange = player.update(dt)
+            const doesPlayerUpdate = player.update(dt)
 
             //Checks for player position changes and updates their position
             if(pchange){ 
```

Comment:
You must also change it here
```suggestion
            if(doesPlayerUpdate){ 
```